src/thresholds.service.js

```diff
@@ -64,14 +64,14 @@ class ThresholdsService {
       index[productId] = Object.keys(weeksOfStock).reduce((productThresholds, threshold) => {
         const level = weeklyLevels[productId] * weeksOfStock[threshold]
         const product = find(products, isId.bind(null, productId))
-        if (!product || !product.presentation) {
-          return productThresholds
+        if (product && product.presentation) {
+          // TODO: product presentations should be ints, not strings
+          const presentation = parseInt(product.presentation, 10)
+          const roundedLevel = Math.ceil(level / presentation) * presentation
+          productThresholds[threshold] = roundedLevel
+        } else {
+          productThresholds[threshold] = level
```

Comment:
oh, just use round to 20.
This is the excel default for all counts, and the highest possible presentation in country.
Will definitely want to create that throw ticket though, after this release, because magic numbers.